__tests__/app.test.js

```diff
@@ -258,26 +258,108 @@ describe('POST: /api/articles/:article_id/comments', () => {
         })
     })
 })
-// describe('PATCH: /api/articles/:article_id', () => {
-//     test('PATCH 200: updates an aticle with a new vote count based a positive incrementation from the body of the request', () => {
-//         const voteIncrement = {inc_votes: 100 }
-//         const exampleArticle =   {
-//             title: "Living in the shadow of a great man",
-//             topic: "mitch",
-//             author: "butter_bridge",
-//             body: "I find this existence challenging",
-//             created_at: 1594329060000,
-//             votes: 100,
-//             article_img_url:
-//               "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-//           }
-//         return request(app)
-//         .patch('/api/articles/1')
-//         .send(voteIncrement)
-//         .expect(200)
-//         .then(({body}) => {
-//             const newVoteCount = exampleArticle.votes + 100
-//             expect(body.votes).toEqual((newVoteCount))
-//         })
-//     })
-// })
\ No newline at end of file
+describe('PATCH: /api/articles/:article_id', () => {
+    test('PATCH 200: updates an aticle with a new vote count based a positive incrementation from the body of the request', () => {
+        const voteIncrement = {inc_votes: 100 }
+        const exampleArticle =   {
+            title: "Living in the shadow of a great man",
+            topic: "mitch",
+            author: "butter_bridge",
+            body: "I find this existence challenging",
+            created_at: 1594329060000,
+            votes: 100,
+            article_img_url:
+              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+          }
+        return request(app)
+        .patch('/api/articles/1')
+        .send(voteIncrement)
+        .expect(200)
+        .then(({body}) => {
+            const newVoteCount = exampleArticle.votes + 100
+            expect(body.votes).toEqual((newVoteCount))
```

Comment:
I get what you're going for here, but you would be quite alright to check the database for the existing votes and say that after your increment that you're expecting a different number.